smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ServiceNamespaceIntegration.kt

```diff
@@ -17,8 +18,8 @@ class ServiceNamespaceIntegration : SwiftIntegration {
         delegator: SwiftDelegator
     ) {
         val service = ctx.settings.getService(ctx.model)
-        val namespaceName = service.nestedNamespaceType(ctx.symbolProvider)
-        val filename = "${ctx.settings.moduleName}/models/$namespaceName.swift"
+        val namespaceName = service.nestedNamespaceType(ctx.symbolProvider).name
+        val filename = "${ctx.settings.moduleName}/${ModelFileUtils.filename(ctx.settings, namespaceName)}"
```

Comment:
You will see several places in this project and in aws-sdk-swift where the `ModelFileUtils.filename()` method is used to resolve the model filename to be used.

I won't comment on all the changes similar to this one.